config/config_spineGeHemis.json

```diff
@@ -0,0 +1,82 @@
+{
+	"command": "train",
+	"gpu": 6,
+	"hdf5_path": "../duke/projects/csf_deepseg/spineGeneric/hdf5/SpineGe.hdf5",
+	"csv_path": "../duke/projects/csf_deepseg/spineGeneric/hdf5/SpineGe.csv",
+	"target_suffix": ["_seg-manual"],
+	"roi_suffix": null,
+	"bids_path": "../duke/projects/csf_deepseg/spineGeneric/result/",
+	"random_seed": 42,
+	"contrast_train_validation": ["T2star", "T2w", "T1w"],
+	"contrast_balance": {},
+	"contrast_test": ["T2star", "T2w", "T1w"],
+	"center_test": [],
+	"retrain_model": null,
+	"out_channel": 1,
+	"depth": 2,
+	"batch_size": 64,
+	"dropout_rate": 0.3,
+	"batch_norm_momentum": 0.1,
+	"num_epochs": 100,
+	"initial_lr": 0.001,
+	"lr_scheduler": {"name": "CosineAnnealingWarmRestarts", "T_0": 10, "T_mult": 1},
+	"loss": {"name": "dice"},
+	"log_directory": "log_large_retrain",
+	"film_layers": [0, 0, 0, 0, 0, 0, 0, 0],
+	"mixup_bool": false,
+	"mixup_alpha": 2,
+	"metadata": "constrast",
+	"multichannel": false,
+	"HeMIS": true,
+	"ram": true,
+	"missing_probability": 0.0001,
+	"target_lst": ["T2w_seg-manual"],
+	"roi_lst": null,
```

Comment:
I am not sure to understand how `roi_lst` is used.

Reply:
target_lst and roi_lst indicates the modality you want to use for the ground truth or for the roi. For example, i'd like to use the roi from T1w modality (sub000_T1w_seg-manual and not sub000_T2w_seg-manual). Target_lst is a list to have the option to have more than one ground truth. Roi_lst is a list since some places deal with roi and gt the same way.